jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/ConfigsAsBooleanTest.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common.util;
+
+import org.eclipse.jkube.kit.common.Configs;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+import static org.assertj.core.api.Assertions.assertThat;
+@RunWith(Parameterized.class)
+public class ConfigsAsBooleanTest {
+
+    @Parameterized.Parameters(name = "{index}: asBoolean({0})={1}")
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] {
+                { " 1 2 1337", false },{ "1", false },{"0",false},{"fALsE", false },{"false", false}
```

Comment:
This looks good! However, I was thinking if we could also add a description parameter field which might improve readability a little.

```java
    @Parameterized.Parameter
    public String description;
```

Then you can provide this in data array like this (basically porting test names here):
```java
    @Parameterized.Parameters(name = "{0}: asBoolean({1})={2}")
    public static Collection<Object[]> data() {
        return Arrays.asList(new Object[][] {
                { "Unsupported String", " 1 2 1337", false },
                { "One", "1", false },
                { "Zero", "0", false },
                { "FalseMixedCase", "fALsE", false },
                { "False", "false", false},
                // ...           
```

Also, you seem to only have cases with value false here. Please port test cases testing true as well i.e `asBooleanValueWithTrueShouldReturnTrue` `asBooleanValueWithTrueUpperCaseShouldReturnTrue` `asBooleanValueWithTrueMixedCaseShouldReturnTrue`

Reply:
added suggested changes, Thanks!